centreon/tests/e2e/features/Cloud-notifications/common.ts

```diff
@@ -180,11 +215,342 @@ const waitUntilLogFileChange = (): Cypress.Chainable => {
+      stalk_on_critical,
+      stalk_on_ok,
+      stalk_on_unknown,
+      stalk_on_warning,
+      state,
+      state_type,
+      volatile
+    ].join('\t');
+
+    centreonStorageServicesValues += values + '\n';
+  }
+  cy.writeFile(
+    './fixtures/notifications/centreon_storage_services.txt',
+    centreonStorageServicesValues
+  );
+  cy.log('Values generated and stored in centreon_storage_services.txt.');
+
+  // Generate values for centreon_services.txt
+  let centreonServicesValues = '';
+  // The first service will got an id of 28
+  for (let i = 28; i < 1028; i++) {
+    const service_id = i;
+    const service_template_model_stm_id = 3;
+    const service_description = `service_${i - 27}`;
+    const service_is_volatile = 2;
+    const service_max_check_attempts = 1;
+    const service_active_checks_enabled = 0;
+    const service_passive_checks_enabled = 1;
+    const service_parallelize_check = 2;
+    const service_obsess_over_service = 2;
+    const service_check_freshness = 2;
+    const service_event_handler_enabled = 2;
+    const service_flap_detection_enabled = 2;
+    const service_process_perf_data = 2;
+    const service_retain_status_information = 2;
+    const service_retain_nonstatus_information = 2;
+    const service_notifications_enabled = 2;
+    const contact_additive_inheritance = 0;
+    const cg_additive_inheritance = 0;
+    const service_inherit_contacts_from_host = 1;
+    const service_use_only_contacts_from_host = 0;
+    const service_locked = 0;
+    const service_register = 1;
+    const service_activate = 1;
+
+    const values = [
+      service_id,
+      service_template_model_stm_id,
+      service_description,
+      service_is_volatile,
+      service_max_check_attempts,
+      service_active_checks_enabled,
+      service_passive_checks_enabled,
+      service_parallelize_check,
+      service_obsess_over_service,
+      service_check_freshness,
+      service_event_handler_enabled,
+      service_flap_detection_enabled,
+      service_process_perf_data,
+      service_retain_status_information,
+      service_retain_nonstatus_information,
+      service_notifications_enabled,
+      contact_additive_inheritance,
+      cg_additive_inheritance,
+      service_inherit_contacts_from_host,
+      service_use_only_contacts_from_host,
+      service_locked,
+      service_register,
+      service_activate
+    ].join('\t');
+
+    centreonServicesValues += values + '\n';
+  }
+  cy.writeFile(
+    './fixtures/notifications/centreon_services.txt',
+    centreonServicesValues
+  );
+  cy.log('Values generated and stored in centreon_services.txt.');
+
+  // Generate values for host_service_relation.txt
+  let hostServiceRelationValues = '';
+  // The first service will got an id of 28
+  for (let i = 28; i < 1028; i++) {
+    const host_host_id = 15;
+    const service_service_id = i;
+
+    const values = `${host_host_id}\t${service_service_id}\n`;
+    hostServiceRelationValues += values;
```

Comment:
```suggestion
    const host_host_id = 15;
    const service_service_id = i;

    const values = `${host_host_id}\t${service_service_id}\n`;
    hostServiceRelationValues += `15\t${i}\n`;
```

Reply:
Using 15 and i directly doesn't make sense for a code reader. I believe declaring constants is mandatory so that everyone can understand the code.